miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultContainerService.java

```diff
@@ -81,6 +81,12 @@ public AuthorizationManager getAuthorizationManager() {
     return authorizationManager.filterUnreadable(containers);
   }
 
+  @Override
+  public Collection<Partition> listPartitionsByPoolId(long poolId) throws IOException {
+    Collection<Partition> containers = containerDao.listAllPartitionsByPoolId(poolId);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "containers". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)